plotly/src/plot.rs

```diff
@@ -818,4 +820,47 @@ mod tests {
+        // Limit the test to the first LEN characters
+        const LEN: usize = 100;
+        assert_eq!(expected[..LEN], image_svg[..LEN]);
```

Comment:
The first 100 characters are mostly (or just) SVG metadata that I suspect will make the test less robust.
Is the reason for this to avoid expensive computation or is it maybe to just test that in fact an SVG was produced?
If the latter, then I totally understand. :)

Reply:
Hi @emilbratt. Magical number indeed 😅. Well, when I added the test , I found out that the SVG ends up containing some element `id`s that are new with each invocation. Didn't have the time to go down the rabbit hole to find out how those get generated. So I ended up limiting the test to the beginning of the string that does not contain any such `id`.  I agree with you, that probably it would be wiser to limit the assert to the first 10ish characters or so. Will probably change it to that before releasing the new version.

Reply:
@andrei-ng Sounds like a good choice to me, hehe. Thank you for taking your time with this.
I was so happy when I read it got merged and am looking forward to the next release. :)